web/client/reducers/controls.js

```diff
@@ -67,7 +67,11 @@ function controls(state = {}, action) {
             })
         });
     case RESET_CONTROLS: {
-        const resetted = Object.keys(state).reduce((previous, controlName) => {
+        let newControls = Object.keys(state);
```

Comment:
There is always a way to use const:
const newControls = Object.keys(state).filter(c => (action.skip || []).indexOf(c) === -1)

Reply:
ok...